testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ExpectedDeprecationWarning.java

```diff
@@ -29,6 +29,9 @@
  */
 public abstract class ExpectedDeprecationWarning {
 
+    private static final Pattern DEPRECATION_WARNING_LOG_PREFIX_PATTERN =
+        Pattern.compile("^.* \\[WARN] \\[org\\.gradle\\.internal\\.featurelifecycle\\.LoggingDeprecatedFeatureHandler] ");
```

Comment:
💭 This may be cleaner if you quote it:
```suggestion
        Pattern.compile("^.* " + Pattern.quote("[WARN] [org.gradle.internal.featurelifecycle.LoggingDeprecatedFeatureHandler] "));
```